### PR TITLE
Hive. ListStore. Add support JSONValue to Method "set"

### PR DESCRIPTION
**Pull request by viktor-liablin**
Hive. ListStore. Add support JSONValue to Method "set"

### CODE REVIEW

File: src/hive/stores/list.js [hunk 1]
```diff
@@ -40,16 +40,26 @@ export class ListStore extends HiveStore {
       })
   }
 
-  set(value, index) {
-    if (Array.isArray(value)) {
+  set(...args) {
```
Comment (Valodya) [inline src/hive/stores/list.js:43]:
```
set(value, index) {
    if (typeof index === 'undefined') {
      ...
```

File: test/tsd.ts [hunk 2]
```diff
@@ -837,7 +837,16 @@ function testHiveStores() {
             promiseListOfJSONValue = Backendless.Hive(str).ListStore(str).get(num, num);
 
             promiseNumber = Backendless.Hive(str).ListStore(str).set([str, str]);
```
Comment (Valodya) [inline test/tsd.ts:839]:
```
promiseNumber = Backendless.Hive(str).ListStore(str).set(str);
promiseNumber = Backendless.Hive(str).ListStore(str).set(num);
...
```

Comment (viktor-liablin) [reply test/tsd.ts:839]: It is not valid.
If only one arg, it should be only array

File: src/hive/stores/list.js (hunk 1 above)
Comment (viktor-liablin) [reply src/hive/stores/list.js]: \+

File: test/tsd.ts (hunk 2 above)
Comment (Valodya) [reply test/tsd.ts:839]: ok

**Pull request merged**